src/components/Swap/index.tsx

```diff
@@ -40,32 +42,36 @@ function getTransactionFromMap(
   return
 }
 
+// SwapProps also currently includes props needed for wallet connection, since the wallet connection component exists within the Swap component
+// TODO(kristiehuang): refactor WalletConnection outside of Swap component
```

Comment:
Is this TODO still relevant?

Reply:
https://github.com/Uniswap/widgets/pull/123 - let me know if this makes sense

Wanted to pull wallet connection out of the Swap component since it's separate concerns -- and also makes it slightly easier to build Buy/Donate/etc widgets in the future